game.py

```diff
@@ -124,10 +125,12 @@ def _all_clear(self):
     def _initiating(self):
         return self._activity._collab.props.leader
 
-    def more_dots(self):
+    def more_dots(self, size=None):
```

Comment:
I think it would be better to set size to 0 and then compare to that as we're dealing with ints and the zero value of ints is 0.

Reply:
Fixed. I thought this was more pythonic.